.github/workflows/lint.yaml

```diff
@@ -1,23 +1,11 @@
 name: lint
 
-# Triggers the workflow on push or pull request events
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Do you need to put any type in `pull_request:`?
https://github.com/marketplace/actions/pull-request-linter

I just felt it is a little weird to me to just see:

```
pull_request:
```

Reply:
I've used the DM-recommended `lint.yaml` https://lsstc.slack.com/archives/C9BEJU1T3/p1667519766208999?thread_ts=1667515906.245289&cid=C9BEJU1T3

Reply:
From https://github.com/lsst/daf_butler/blob/main/.github/workflows/lint.yaml